### PR TITLE
added CardUI

### PR DESCRIPTION
**Pull request by alexgul1**

### CODE REVIEW

File: src/js/CardUI.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    this.description = description;
+  }
+
+  renderCard(target) {
+    const card = document.createElement('li');
+    card.classList.add('card');
+
+    const title = document.createElement('p');
+    title.classList.add('title');
+    title.innerHTML = this.title;
```
Comment (matroskin062) [inline src/js/CardUI.js]: It's better to use `textContent` when you're working with straight text. Moreover, `textContent` works faster due to `innerHTML` parses content as HTML.

Comment (muartem) [reply src/js/CardUI.js]: innerText maybe?

Comment (alexgul1) [reply src/js/CardUI.js]: > It's better to use `textContent` when you're working with straight text. Moreover, `textContent` works faster due to `innerHTML` parses content as HTML.

I agree, corrected

**Comment by alexgul1** [comment]
added deleteCard handler

**Review by matroskin062 — APPROVED** [review]

**Pull request merged**